### ISSUE TITLE
Invoke-CrmAction throws an Error on [Microsoft.Xrm.Sdk.OrganizationRequest] constructor  

### ISSUE THREAD

**Issue by moritzwald** [seanmcne/Microsoft.Xrm.Data.PowerShell#275]
Hi, 

I am working in PS4 and getting following error on Invoke-CrmAction call. SDK dlls are updated as in latest xrm data powershell module release.
Invoke-CrmAction : Method invocation failed because [Microsoft.Xrm.Sdk.OrganizationRequest] does not contain a method named 'new'.
[error]At E:\Builds\agent\_work\Reset.ps1:80 char:8
[error]+        Invoke-CrmAction -conn $CRMConn -Name 'Test' -Verbose
[error]+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[error]    + CategoryInfo          : InvalidOperation: (:) [Invoke-CrmAction], RuntimeException
[error]    + FullyQualifiedErrorId : MethodNotFound,Invoke-CrmAction
[error] 
[error]Process completed with exit code 1 and had 1 error(s) written to the error stream.

**Comment by seanmcne** [comment]
I think we could probably look to flip over from: 
```powershell
$request = [Microsoft.Xrm.Sdk.OrganizationRequest]::new($Name)
```
To something like: 
```powershell
$request = new-object Microsoft.Xrm.Sdk.OrganizationRequest
$request.RequestName = $Name
```

**Comment by seanmcne** [comment]
Will be changed in release 2.7.

**Comment by seanmcne** [comment]
Fixed in 2.7.1 which released earlier today.
